bootstrap-sass/src/test/java/de/agilecoders/wicket/sass/BootstrapSassTest.java

```diff
@@ -61,7 +62,7 @@ void importWebContext() throws Exception {
 
         SassSource sassSource = sass.getSassContext(res, null);
         String css = sass.getCss(sassSource);
-        assertThat(css, is(equalTo(".rule {\n  background: #999; }\n")));
+        assertThat(css, is(equalTo(".rule {" + lineSeparator() + "  background: #999; }" + lineSeparator())));
```

Comment:
Using `lineSeparator()` looks better than the hardcoded `/`s in UrlImporter

Reply:
The hardcoded "/" is there because it is a URL, similar to https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/wicket-9.x-bootstrap-4.x/bootstrap-sass/src/main/java/de/agilecoders/wicket/sass/UrlImporter.java#L209